### ISSUE TITLE
Module Admin - Database is created in wrong team

### ISSUE THREAD

**Issue by rompik** [eryar/PipeCAD#18]

**Comment by eryar** [comment]
PipeCad.CreateItem(theType, theName)  is general method, and only used for item creation, donot check item name.
I think it's better to check Team name in Python side.

**Comment by rompik** [comment]
Sorry, but I think this approach can bring more issues than benefits. For admin elements creation, it has to be used additional checks inside Core of software to prevent users create wrong elements or use incorrect hierarchy. Will be better if core will contain separate method for creating main project elements: User, Team, Database, Mdb. In this case it will allow avoiding user to make mistakes during setup admin part of project and also will simplify a little Python code, for ex.:   
I need to create Database using script and I will use next code:
```python 
PipeCad.SetCurrentItem("/*PIPE")                    # I have to be sure that will not mistake name of Team for which I want to create db
Dbli = PipeCad.CurrentItem().Member[0]
PipeCad.SetCurrentItem(Dbli)
PipeCad.CreateItem("DB", "*PIPE/DESI-X-XXX-001")  # In db name I need to use *CS to have correct name 
aDbItem = PipeCad.CurrentItem()
aDbItem.Description = "Db for module X-XXX-001 - Piping Discipline"
aDbItem.DbType = "DESI"
aDbItem.DbNumber = 2000
aDbItem.CreateDbElement("X-XXX-001/PIPE")
 ```
In case if you create dedicated methods:
```python 
PipeCad.CreateDb("PIPE/DESI-X-XXX-001", "DESI", 2000, "100",  "Db for module X-XXX-001 - Piping Discipline")  # Team / DbName , DbType, DbNumber, DbArea, DbDescription
aDbItem = PipeCad.CurrentItem()
aDbItem.CreateDbElement("X-XXX-001/PIPE")
 ```
Using last code system will automatically check correct hierarchy place depends on Team and use correct name (without remembering to add * before name of Team in db name). Also, it will allow reducing number of lines of code, that could be also good benefit. 

So I see next syntax of methods for Admin elements:
```Python
PipeCad.CreateTeam("PIPE", "Team for piping department") # Team Name, Team Description
PipeCad.CreateUser("JBROWN", "James BROWN") # User login, User Description
PipeCad.CreateDb("PIPE/DESI-X-XXX-001", "DESI", 2000, "100",  "Db for module X-XXX-001 - Piping Discipline" ) # Team / DbName , DbType, DbNumber, DbArea, DbDescription
PipeCad.CreateMdb("X-XXX-001", "Mdb for scope of module X-XXX-001") # Mdb Name, Mdb Description
```

**Comment by eryar** [comment]
Yes, I agree with you.
Add dedicated methods is good, I will add those methods to PipeCad.

**Comment by eryar** [comment]
Fixed PipeCAD-1.0.10
